docs/source/development/data_storage.rst

```diff
@@ -0,0 +1,24 @@
+.. _core-development:data-storage:
+
+*************
+Data storage
+*************
+
+
+.. _core-development:orm:database:
+
+Database schema & migrations
+=============================
+
+
```

Comment:
All missing sections should contain links to an associated issue.

Reply:
Ok